gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/PermissionService.groovy

```diff
@@ -93,7 +93,7 @@ class PermissionService {
     if (fiatStatus.isEnabled()) {
       HystrixFactory.newVoidCommand(HYSTRIX_GROUP, "sync") {
         try {
-          fiatService.sync()
+          fiatService.sync(new ArrayList())
```

Comment:
Mmm, maybe `Collections.emptyList()` instead so we don't need to be creating new objs?

Reply:
Fixed